string keyword
string answer
uint8 action
uint8 action_mode
---
uint8 error_code
